spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -929,12 +934,6 @@ function timeoutOnClose(
     // delete our commitment
     provableStore.delete(packetCommitmentPath(packet.sourcePort, packet.sourceChannel, packet.sequence))
 
-    if channel.order === ORDERED {
```

Comment:
what does this change have to do with the new channel type?

Reply:
This is just in the wrong place originally.

This is pseudocode for when an ORDERED channel times out. But that is already specified on line 856.

But this function is for when a channel has already closed, and we want to time out all pending packets. So there is no  special case for ORDERED channels here.